readthedocs/templates/projects/project_create.html

```diff
@@ -1,23 +1,6 @@
 {% extends "dashboard/base_dashboard.html" %}
 {% load i18n %}
 
-{% block extra_links %}
-  <link rel="stylesheet" type="text/css" href="{{ MEDIA_URL }}lib/markitup/skins/simple/style.css" />
-  <link rel="stylesheet" type="text/css" href="{{ MEDIA_URL }}lib/markitup/sets/sphinx/editor.css" />
-{% endblock %}
-
-{% block dash-nav-create %}active{% endblock %}
```

Comment:
This is a required line.

Reply:
Sorry, it was removed by mistake. Thanks for pointing out.